pulseapi/entries/test_models.py

```diff
@@ -0,0 +1,68 @@
+import factory
+from faker import Factory as FakerFactory
+
+from pulseapi.entries.models import Entry
+from pulseapi.users.test_models import EmailUserFactory
+
+
+faker = FakerFactory.create()
+
+
+class EntryFactory(factory.DjangoModelFactory):
+
+    # def __init__(self, tags=['foo','bar']):
+    #     self.tags = tags
+    #     print(self.tags)
+
+    title = factory.LazyAttribute(lambda o: 'title '+' '.join(faker.words(nb=1)))
```

Comment:
we probably want a few more than just one word for title, especially since we restricted it to only 140 characters